sdk/java-sdk-spring/src/test/java/kalix/spring/ComponentClientTest.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.spring;
+
+import com.google.protobuf.any.Any;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.DynamicMessage;
+import com.google.protobuf.InvalidProtocolBufferException;
+import kalix.javasdk.JsonSupport;
+import kalix.javasdk.impl.AnySupport;
+import kalix.javasdk.impl.ComponentDescriptor;
+import kalix.javasdk.impl.JsonMessageCodec;
+import kalix.javasdk.impl.RestDeferredCall;
+import kalix.spring.impl.RestKalixClientImpl;
+import kalix.spring.testmodels.Message;
+import kalix.spring.testmodels.action.ActionsTestModels.GetClassLevel;
+import kalix.spring.testmodels.action.ActionsTestModels.GetWithOneParam;
+import kalix.spring.testmodels.action.ActionsTestModels.GetWithoutParam;
+import kalix.spring.testmodels.action.ActionsTestModels.PostWithOneQueryParam;
+import kalix.spring.testmodels.action.ActionsTestModels.PostWithTwoParam;
+import kalix.spring.testmodels.action.ActionsTestModels.PostWithoutParam;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+class ComponentClientTest {
```

Comment:
I can run this test from intellij, but I'm not sure how to enable it during `sbt test` run.